test/Auth/authModule.js

```diff
@@ -2,27 +2,39 @@ import test from 'ava';
 import { actionTest, reducerTest } from 'redux-ava';
 
 import {
+  CHANGE_USERNAME,
   changeUsername,
+  LOGIN_REQUEST,
   loginRequest,
   reducer,
 } from '../../src/Auth/authModule';
 
 const username = 'Alice';
+const theOtherUsername = 'Bill';
 
-test('Action: changeUsername', actionTest(
+test('Action: changeUsername: Alice', actionTest(
```

Comment:
This test description doesn't really help in terms of understanding what's the purpose of this test. Same as the next test.